docs/user-guide/ci.rst

```diff
@@ -0,0 +1,62 @@
+CI Configuration
+================
+
+Follow these steps to configure CI for GitHub Actions on your repository:
+
+* Step 1:
+   Add the certora cli key `as a secret`_ to your repository, the secret name is "CERTORAKEY"
+   and the value is the key provided.
+
+* Step 2:
+   Create a "certora_verification.yml" file under ".github" directory and 
+   use the following .yaml example as a template you can use to start running certora cli.
+
+.. code-block:: yaml
+
+   name: certora-verification
+
+   on:
+   push:
+      branches:
+         - main
+   pull_request:
+      branches:
+         - main
+
+   workflow_dispatch:
+
+   jobs:
+   verify:
+      runs-on: ubuntu-latest
+
+      steps:
+         - uses: actions/checkout@v4
+         with:
+            submodules: recursive
+
+         - name: Install python
+         uses: actions/setup-python@v2
+         with: { python-version: 3.9 }
+
+         - name: Install java
+         uses: actions/setup-java@v1
+         with: { java-version: "11", java-package: jre }
+
+         - name: Install certora cli
+         run: pip3 install certora-cli==7.0.7
+
+         - name: Install solc
+         run: |
+            wget https://github.com/ethereum/solidity/releases/download/v0.8.23/solc-static-linux
+            chmod +x solc-static-linux
+            sudo mv solc-static-linux /usr/local/bin/solc8.23
+
+         - name: Verify
+         env:
+            CERTORAKEY: ${{ secrets.CERTORAKEY }}
+         run: |
+            # Add your code here
```

Comment:
This is more of @shoham-certora 's concern,
but do we want to explain how to add conf files to the list?

Reply:
i'm not sure, how do we usually add it ?

Reply:
We just need an example like the one I've sent you.

Reply:
@shoham-certora could you add an example of running a conf file instead of "# Add your code here" ?

Reply:
A simple example would be 
```yaml
        run: |
          certoraRun path_to_conf_file.conf
```
A more complicated one:
```yaml
         - name: Verify ${{ matrix.rule }}
         env:
            CERTORAKEY: ${{ secrets.CERTORAKEY }}
         run: 
            certoraRun  ${{ matrix.rule }}

    strategy:
       fail-fast: false
       max-parallel: 16
       matrix:
          rule:
             - path-to-conf1.conf --rule rule1  # run specific rule
             - path-to-conf1.conf --rule rule2
             - path-to-conf2.conf  # run the entire conf
```